test/Bugs/Issue5837.err

```diff
@@ -1,7 +1,7 @@
 AGDA_FAILURE
 
 ret > ExitFailure 42
-out > Issue5837.agda:18,13-17
+out > Issue5837.agda:18,13-17: error: [GenericDocError]
```

Comment:
I think we should avoid printing `GenericError` and `GenericDocError` to the user, if at all avoidable. Worst case, we should perhaps have a special case to just print no tag.

Reply:
Maybe extra motivation to get rid of GenericError?

We could turn off printing it, but then we'd lose incentive to fix it...

Reply:
I would turn off printing `GenericError` just before the release of 2.8.0, but keep it on master to incentivise the removal of `GenericError`s.